model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/validation/DeviceTokenValidator.java

```diff
@@ -72,6 +78,8 @@ public boolean isValid(Installation installation, ConstraintValidatorContext con
                 return WINDOWS_DEVICE_TOKEN.matcher(deviceToken).matches();
             case SIMPLE_PUSH:
                 return SIMPLE_PUSH_DEVICE_TOKEN.matcher(deviceToken).matches();
+            case ADM:
+                return ADM_DEVICE_TOKEN.matcher(deviceToken).matches();
```

Comment:
Not sure when it was introduced but I have this also on master , tracked by https://issues.jboss.org/browse/AGPUSH-1272